app/components/sy-checkmark/template.hbs

```diff
@@ -1 +1 @@
-<FaIcon ...attributes @icon={{this.icon}} />
+<FaIcon class="{{if @checked "blue-checked"}}" ...attributes @icon={{this.icon}} />
```

Comment:
Getting closer! Please decouple the *blue highlighting* from the `checked` parameter. You could name it `highlight`. And if `highlight` is truthy, then dye it blue. This way we can highlight the checkmarks in the activity tab, but won't influence all the other occurrences of the component.